pkg/crypto/clef/clef.go

```diff
@@ -30,7 +32,13 @@ type ExternalSignerInterface interface {
 	Accounts() []accounts.Account
 }
 
+// ExternalSignerInterface is the interface for rpc.RpcClient
+type RpcClient interface {
```

Comment:
consider interface name to be `Client`, or `Caller` (more "go")

Reply:
went with `Client`